hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/validation/ValidatorRegistry.java

```diff
@@ -178,13 +179,8 @@ private EnumMap<RequestProcessingPhase, List<Method>> newPhaseMap() {
     return new EnumMap<>(RequestProcessingPhase.class);
   }
 
-  private <K, V> V getAndInitialize(K key, V defaultValue, Map<K, V> from) {
-    V inMapValue = from.get(key);
-    if (inMapValue == null || !from.containsKey(key)) {
-      from.put(key, defaultValue);
-      return defaultValue;
-    }
-    return inMapValue;
+  private <K, V> V getAndInitialize(K key, Supplier<V> defaultSupplier, Map<K, V> from) {
+    return from.computeIfAbsent(key, k -> defaultSupplier.get());
```

Comment:
Probably we can just inline this method, I am not sure why I did not used this approach before, but this is a great simplification here.

Reply:
Yes, it can be inlined.  I kept it to make the change easier to see.